# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-04-10 12:50+0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: account/views.py:70
msgid "Username and password cannot be empty"
msgstr "旧密码不能为空"

#: account/views.py:78
msgid "User does not exist.Please register an account first"
msgstr "帐号不存在，请先注册帐号"

#: account/views.py:93 account/views.py:484
msgid "Something went wrong. Please try again"
msgstr "错误，请再尝试一次"

#: account/views.py:99 account/views.py:490
msgid "Username or password is not correct"
msgstr "帐号或密码不正确"

#: account/views.py:110 account/views.py:506
msgid "You have successfully logged out"
msgstr "已成功登出"

#: account/views.py:142 account/views.py:336
msgid "Invalid email address"
msgstr "信箱格式不正確"

#: account/views.py:167
msgid ""
"User has successfully created. Please check email for account validation"
msgstr "已经成功建立使用者，请进行信箱验证"

#: account/views.py:171
msgid ""
"User has successfully created. Email sent failed. Please resend the email to "
"validate the account"
msgstr "已经成功建立使用者，信箱传送失败，请再次传送验证信"

#: account/views.py:174
msgid "User name has existed"
msgstr "使用者名称已存在"

#: account/views.py:193
msgid "Please register an account first"
msgstr "请先注册帐号"

#: account/views.py:196 account/views.py:259
msgid "Account has been confirmed. No need to get confirmation email again"
msgstr "帐号已验证，不需再次传送验证信"

#: account/views.py:208
msgid "Please wait for a while to resend the code"
msgstr "请稍候再尝试"

#: account/views.py:226
msgid "Confirmation email has been sent"
msgstr "已寄送验证信"

#: account/views.py:228
msgid "Failed to send email"
msgstr "传送信件失败"

#: account/views.py:253
msgid "Invalid code"
msgstr "认证失败"

#: account/views.py:266
msgid "User name has been changed"
msgstr "帐号已更改"

#: account/views.py:270
msgid "Account has been validated"
msgstr "帐号已验证"

#: account/views.py:273
msgid "Account validation failed"
msgstr "帐号验证失败"

#: account/views.py:324 account/views.py:403 account/views.py:431
#: account/views.py:551 account/views.py:596 account/views.py:634
#: account/views.py:685 chatbot/views.py:179 chatbot/views.py:207
#: chatbot/views.py:237 chatbot/views.py:288 chatbot/views.py:301
#: chatbot/views.py:320 chatbot/views.py:330 chatbot/views.py:377
#: chatbot/views.py:390 chatbot/views.py:409 chatbot/views.py:419
#: faq/views.py:107 faq/views.py:179 faq/views.py:191 faq/views.py:213
#: faq/views.py:223 faq/views.py:284 faq/views.py:296 faq/views.py:319
#: faq/views.py:329 faq/views.py:468 history/views.py:57 report/views.py:85
#: taskbot/views.py:88 taskbot/views.py:123 taskbot/views.py:153
msgid "Not found"
msgstr "资料不存在"

#: account/views.py:340
msgid "Username has existed.Please use other one"
msgstr "帐号名称已存在，请输入其他名称"

#: account/views.py:349
msgid "Sending confirmation email failed"
msgstr "寄送认证信失败"

#: account/views.py:357
msgid "Confirmation email has sent.Please check your mailbox and login again"
msgstr "认证信已寄送，请至信箱进行认证后登入"

#: account/views.py:367
msgid "Old password cannot be empty"
msgstr "旧密码不能为空"

#: account/views.py:373 account/views.py:564
msgid "Old password is not correct"
msgstr "旧密码不正确"

#: account/views.py:378 account/views.py:562
msgid "Password has updated.Please login again"
msgstr "密码已更新，请重新登入"

#: account/views.py:392
msgid "Please select one language"
msgstr "请选择语言"

#: account/views.py:395 account/views.py:588 chatbot/views.py:193
#: chatbot/views.py:336 chatbot/views.py:425 faq/views.py:227 faq/views.py:333
#: taskbot/views.py:110
msgid "Update succeeded"
msgstr "更新成功"

#: account/views.py:398 account/views.py:447 account/views.py:544
#: account/views.py:650 account/views.py:726 chatbot/views.py:168
#: chatbot/views.py:195 chatbot/views.py:250 chatbot/views.py:338
#: chatbot/views.py:427 faq/views.py:204 faq/views.py:229 faq/views.py:309
#: faq/views.py:335 faq/views.py:413 paidtype/views.py:90 taskbot/views.py:79
#: taskbot/views.py:112 taskbot/views.py:166
msgid "No content"
msgstr "无内容"

#: account/views.py:408 account/views.py:601 chatbot/views.py:210
#: taskbot/views.py:126
msgid "Please confirm the deletion first"
msgstr "请先确认是否删除"

#: account/views.py:415 account/views.py:608
msgid "Deleting account failed"
msgstr "删除帐号失败"

#: account/views.py:437 account/views.py:640 chatbot/views.py:241
#: taskbot/views.py:157
msgid "Please enter the password"
msgstr "请输入密码"

#: account/views.py:443 account/views.py:646 chatbot/views.py:246
#: taskbot/views.py:162
msgid "Delete confirmed"
msgstr "确认删除"

#: account/views.py:445 account/views.py:648 chatbot/views.py:248
#: taskbot/views.py:164
msgid "Password is not correct"
msgstr "密码不正确"

#: account/views.py:467
msgid "User does not exist. Please contact admin."
msgstr "帐号不存在，请联系客服人员"

#: account/views.py:471
msgid "Only staff can login this page"
msgstr "必须为管理员帐号"

#: account/views.py:540
msgid "Create new agent succeeded"
msgstr "创建管理员帐号成功"

#: account/views.py:542 faq/views.py:202 faq/views.py:307 history/views.py:128
msgid "Create failed"
msgstr "创建失败"

#: account/views.py:572
msgid "User name existed"
msgstr "使用者名称已存在"

#: account/views.py:582
msgid "Do not support this language"
msgstr "不支援此语言"

#: account/views.py:593 account/views.py:630
msgid "Superuser cannot be deleted"
msgstr "不能删除超级管理员帐号"

#: account/views.py:699
msgid "Invalid type"
msgstr "方案不正确"

#: account/views.py:722
msgid "Update successed"
msgstr "更新成功"

#: account/views.py:724
msgid "paid_type cannot be empty"
msgstr "方案不能为空"

#: chatbot/views.py:116
msgid "Reach bot create limitation"
msgstr "已到建立机器人数量上限"

#: chatbot/views.py:128
msgid "Reach faq create limitation"
msgstr "已到问答数量上限"

#: chatbot/views.py:159 taskbot/views.py:70
msgid "Create bot failed. Cause by NLU error."
msgstr "建立机器人失败"

#: chatbot/views.py:166 taskbot/views.py:77
msgid "Create bot failed"
msgstr "建立机器人失败"

#: chatbot/views.py:218 taskbot/views.py:134
msgid "Deleting bot failed"
msgstr "删除机器人失败"

#: chatbot/views.py:292 chatbot/views.py:305 chatbot/views.py:324
msgid "Not allowed to get user line data"
msgstr "不允许此操作"

#: chatbot/views.py:381 chatbot/views.py:394 chatbot/views.py:413
msgid "Not allowed to get user fb data"
msgstr "不允许此操作"

#: faq/views.py:117
msgid "Faq group exceeded upper limit"
msgstr "问答组已到数量上限"

#: faq/views.py:126
msgid "Create faq group failed"
msgstr "创建问答组失败"

#: faq/views.py:363 faq/views.py:375
msgid "File type is not correct. Should be type utf8 or big5."
msgstr "格式不正确，需为utf8或big5"

#: faq/views.py:371
msgid "CSV column is not correct"
msgstr "CSV 栏位数量不正确"

#: faq/views.py:400
msgid "Group over limitation"
msgstr "问答组超过上限"

#: faq/views.py:408
msgid "Upload succeeded"
msgstr "上传成功"

#: faq/views.py:411 faq/views.py:454
msgid "Bot not found"
msgstr "机器人不存在"

#: faq/views.py:474
msgid "Training bot succeeded"
msgstr "成功训练机器人"

#: history/views.py:100
msgid "Chatbot not found"
msgstr "机器人不存在"

#: history/views.py:107
msgid "Matching result not found"
msgstr "推荐纪录不存在"

#: history/views.py:114
msgid "FAQ group does not exisit"
msgstr "问答组不存在"

#: history/views.py:125
msgid "Create new question successed"
msgstr "成功建立新问题"

#: paidtype/views.py:67 paidtype/views.py:100
msgid "Not allowed"
msgstr "不允许此操作"

#: paidtype/views.py:82
msgid "Create paidtype failed"
msgstr "建立方案失败"

#: paidtype/views.py:106
msgid "Cannot delete this paidtype"
msgstr "无法删除这个方案"

#: paidtype/views.py:111
msgid "User is still using this paidtype"
msgstr "还有使用者使用这个方案"

#: taskbot/views.py:97
msgid "Key missing or empty: robot_name"
msgstr "缺少robot_name"

#~ msgid "English"
#~ msgstr "英文"

#~ msgid "Simplify Chineses"
#~ msgstr "简体中文"

#~ msgid "Traditional Chineses"
#~ msgstr "繁体中文"
